Source/WebGPU/WGSL/AST/ASTValue.h

```diff
@@ -27,32 +27,28 @@
 
 #include "ASTNode.h"
 
-#include <wtf/TypeCasts.h>
-#include <wtf/UniqueRefVector.h>
-
 namespace WGSL::AST {
 
-class Decl : public Node {
+class Value : public Node {
```

Comment:
Same comment as before, seems like this node doesn't need to exist, unless I missed something.